src/Phinx/Migration/Migration.template.php.dist

```diff
@@ -24,8 +24,10 @@ class $className extends $baseClassName
      *
      * Remember to call "create()" or "update()" and NOT "save()" when working
      * with the Table class.
+     *
+     * @param string $direction
      */
```

Comment:
I'd add some description here along the lines that the value will be one of the constants I previously highlighted.